python/cuml/linear_model/logistic_regression.pyx

```diff
@@ -314,6 +336,64 @@ class LogisticRegression(Base):
         """
         return self.qn.predict(X, convert_dtype=convert_dtype)
 
+    def predict_proba(self, X, convert_dtype=False):
+        """
+        Predicts the class probabilities for each class in X
+
+        Parameters
+        ----------
+        X : array-like (device or host) shape = (n_samples, n_features)
+            Dense matrix (floats or doubles) of shape (n_samples, n_features).
+            Acceptable formats: cuDF DataFrame, NumPy ndarray, Numba device
+            ndarray, cuda array interface compliant array like CuPy
+
+        convert_dtype : bool, optional (default = False)
+            When set to True, the predict method will, when necessary, convert
+            the input to the data type which was used to train the model. This
+            will increase memory used for the method.
+        Returns
+        ----------
+        y: array-like (device)
+           Dense matrix (floats or doubles) of shape (n_samples, n_classes)
+        """
+        scores = cp.asarray(self.decision_function(X,
+                            convert_dtype=convert_dtype), order='F').T
+        if self._num_classes == 2:
+            proba = cp.zeros((scores.shape[0], 2))
+            proba[:, 1] = scores.ravel()
+            proba = 1 / (1 + cp.exp(-proba))
```

Comment:
why not combine these two: ```
 proba[:, 1] = 1 / (1 + cp.exp(-scores.ravel()))
```
 that would seem a little clearer (and actually would do ever so slightly less work)